demo/index.html

```diff
@@ -1,81 +1,758 @@
-            <header>Input</header>
-            <input
-              title="input"
-              type="text"
-              placeholder="input"
-              autocomplete="none"
-            />
-            <input title="check" type="checkbox" />
-          </div>
-
-          <div class="column" draggable="true">
-            <header>TextArea</header>
-            <textarea title="textarea" placeholder="textarea"></textarea>
-          </div>
-
-          <div class="column" draggable="true">
-            <header>Select</header>
-            <select title="select">
-              <option>one</option>
-              <option>two</option>
-              <option>three</option>
-            </select>
-            <select title="multi" multiple>
-              <option>one</option>
-              <option>two</option>
-              <option>three</option>
-            </select>
-          </div>
-
-          <div class="column" draggable="true">
-            <header>Image</header>
-            <img
-              src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/grapefruit-slice-332-332.jpg"
-              alt="grapefruit"
-              draggable="false"
-            />
-          </div>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta
+            name="description"
+            content="A polyfill that enables HTML5 drag drop support on mobile (touch) devices."
+        />
+        <meta name="keywords" content="html5, drag, drop, mobile, touch, polyfill" />
+        <title>DragDropTouch</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" href="https://fonts.cdnfonts.com/css/cascadia-code" />
+        <link rel="stylesheet" href="./index.css" />
+        <script src="../dist/drag-drop-touch.esm.min.js?autoload" type="module"></script>
+        <script src="./index.js" type="module" async></script>
+    </head>
+    <body style="display: contents">
+        <div id="container">
+            <header id="navbar">
+                <a href="https://github.com/drag-drop-touch-js/dragdroptouch" target="_blank">
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        width="24"
+                        height="24"
+                        viewBox="0 0 24 24"
+                        fill="none"
+                        stroke="currentColor"
+                        stroke-width="2"
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
```

Comment:
Let's not use inline styling - when used as a regular HTML5 element any SVG element is subject to CSS just like every else (but require you to set the SVG-specific CSS properties like `stroke` or `fill` rather than `color` and `background` of course)